src/main/kotlin/no/nav/k9/los/nyoppgavestyring/mottak/oppgave/OppgaveV3Tjeneste.kt

```diff
@@ -29,9 +31,21 @@ class OppgaveV3Tjeneste(
             oppgavetypeRepository.hent(område, tx).oppgavetyper.find { it.eksternId.equals(oppgaveDto.type) }
                 ?: throw IllegalArgumentException("Kan ikke legge til oppgave på en oppgavetype som ikke er definert")
 
-        oppgavetype.valider(oppgaveDto)
+        oppgavetype.validerInnkommendeOppgave(oppgaveDto)
+
+        val aktivOppgaveVersjon = oppgaveV3Repository.hentAktivOppgave(oppgaveDto.id, oppgavetype, tx)
+        var innkommendeOppgave = OppgaveV3(oppgaveDto, oppgavetype)
+
+        innkommendeOppgave.oppgavetype.oppgavefelter.forEach { oppgavefelt ->
```

Comment:
Både oppdateringen og Feltutleder har nok logikk til å være gode kandidater for tester

Reply:
Har laget enhetstester på feltutleder nå. Hva er "oppdateringen"? Tenkte uansett at det er greit å få dette ut, og så følge opp enhetstester i neste featurebranch.